packages/core/src/Stores/client-store.js

```diff
@@ -1169,6 +1177,9 @@ export default class ClientStore extends BaseStore {
                 // So it will send an authorize with the accepted token, to be handled by socket-general
                 await BinarySocket.authorize(client.token);
             }
+            if (redirect_url) {
+                window.location.replace(routes[redirect_url]);
```

Comment:
i think we can make it more seamless by using history from react router dom

Reply:
As react-router v4, we don't have access to the `history` object since BrowserRouter creates history internally to avoid usage of the singleton pattern.